app/bundles/CoreBundle/Controller/AbstractStandardFormController.php

```diff
@@ -306,7 +306,7 @@ protected function deleteStandard(Request $request, $objectId)
 
             $model->deleteEntity($entity);
 
-            $identifier = $this->translator->trans($entity->getName());
```

Comment:
`getName()` to `getBlockPrefix()` is enforced by rector, but I can't judge whether thats the right thing (I am no symfony expert)